examples/trials/efficientnet/.gitignore

```diff
@@ -0,0 +1 @@
+EfficientNet-PyTorch
```

Comment:
?

Reply:
EfficientNet-PyTorch is with Apache license and cannot be included into our repo. So user has to clone this repo ahead. Add it to gitignore to prevent this directory to be included in our repo.